ui/raidboss/data/06-ew/trial/golbez-ex.txt

```diff
@@ -1,6 +1,372 @@
 ### The Voidcast Dais (Extreme)
+# -ii 84B5 84B7 849E 84A8 84AA 8481 86B2
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+# Exclude the first auto since it's delayed wildly depending on tank movement
+9.4 "--sync--" sync / 14:[^:]*:Golbez:8464:/ window 15,15
+12.4 "Terrastorm (cast)" sync / 1[56]:[^:]*:Golbez:8464:/
```

Comment:
Yeah I am iffy on things like `(cast)` especially if it's not a thing you would mitigate, but these are also 9 seconds apart, so it's not like there's much else going on, so seems fine enough.

Reply:
This is the less egregious instance, compared to the instances of `Phases of the Blade` for instance. But the timing seemed useful for snapshotting for me 🤷